src/formfield/index.tsx

```diff
@@ -20,11 +24,13 @@ export const FormField = createComponent<FormFieldProps>(function FormField(
 ) {
   useFormfieldFoundation(props);
 
-  const { alignEnd, foundationRef, ...rest } = props;
+  const { alignEnd, foundationRef, noWrap, spaceBetween, ...rest } = props;
   const className = useClassNames(props, [
     'mdc-form-field',
     {
-      'mdc-form-field--align-end': props.alignEnd
+      'mdc-form-field--align-end': props.alignEnd,
```

Comment:
Why do you access the properties via props when you have destructured them in line 27?